Formula/miranda.rb

```diff
@@ -0,0 +1,63 @@
+class Miranda < Formula
+  desc "Lazy, purely functional programming language"
+  homepage "http://miranda.org.uk/"
+  url "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-2066-src.tgz"
+  version "2.066"
+  sha256 "521c281e8c2fde87a2cd7c41d9677daa09514debb71435efc08ff3a7c20016eb"
+
+  depends_on "byacc" => :build
+
+  def install
+    # Make tasks are required to run sequentially
+    ENV.deparallelize
+
+    # Makefile assumes that provided paths exist
+    bin.mkpath
+    lib.mkpath
+    man1.mkpath
+
+    miralib = "#{lib}/miralib"
+
+    args = %W[
+      CC=#{ENV.cc}
+      CFLAGS=-O
+      BIN=#{bin}
+      LIB=#{lib}
+      MAN=#{man1}
+    ]
+
+    inreplace "Makefile" do |s|
+      # Skip compilation of `stdenv.m` & provided examples
+      s.gsub! " exfiles", ""
+      # Do NOT touch `#{lib}/miralib/` permissions
+      s.gsub! "./protect", "/usr/bin/true"
+      s.gsub! "./unprotect", "/usr/bin/true"
+      # Do NOT change `#{lib}/miralib/` ownership
+      s.gsub! "./ugroot", "whoami"
+    end
+
+    system "make", "cleanup"
+    system "make", "install", *args
+
+    # Remove compiled files, if any
+    rm_f "#{miralib}/preludx"
+    rm Dir["#{miralib}/**/*.x"]
```

Comment:
Why is this needed?

Reply:
To reassure no compiled files have been put inside the bottle. It should be sufficient to simply call [`make cleanup`](https://github.com/rljacobson/Miranda/blob/49d6dcc/Makefile#L44-L49), which is already done, but I'm using this an additional safety measure. Again, if you bottle those while altering creation timestamps on accompanying sources you'll get runtime error on the first run of Miranda environment.

Reply:
Why can't compiled file be put inside the bottle?

Reply:
Because of https://github.com/Homebrew/homebrew-core/pull/51003#issuecomment-593660181